scripts/netlify.sh

```diff
@@ -15,10 +15,10 @@ yarn
 yarn build
 
 # build examples
-SKIP_PACKAGE_BUILD=true yarn examples:ecommerce:build
-SKIP_PACKAGE_BUILD=true yarn examples:router:build
-SKIP_PACKAGE_BUILD=true yarn examples:media:build
-SKIP_PACKAGE_BUILD=true yarn examples:storybook:build
+SKIP_PACKAGE_BUILD=true yarn examples:ecommerce:build &> /dev/null
```

Comment:
`&> /dev/null` is a temporary workaround.
Without it, the build time exceeded 20 minutes.
How can we shorten the build time without removing build log like this?

Reply:
There seems to be a `--no-progress` option in `ng build`: https://github.com/angular/angular-cli/issues/5962#issuecomment-294150054.

Reply:
Thanks. It worked! https://github.com/algolia/angular-instantsearch/pull/617/commits/88c047d6bf36490b208565902bd73cc11d2d8883